packages/node_modules/@node-red/editor-client/locales/en-US/editor.json

```
@@ -643,6 +643,7 @@
             "errors": {
                 "catalogLoadFailed": "<p>Failed to load node catalogue.</p><p>Check the browser console for more information</p>",
                 "installFailed": "<p>Failed to install: __module__</p><p>__message__</p><p>Check the log for more information</p>",
+                "installTimeout": "<p>Install continuing the background, Nodes will appear in palette when complete.</p><p>Or you can monitor the install logs</p>",
```

```suggestion
                "installTimeout": "<p>Install continuing the background.</p><p>Nodes will appear in palette when complete. Check the log for more information.</p>",
```

I may be wrong, but I don't think we use 'you' in our messages. Have tweaked the wording to be a bit more inline with the other messages.